### ISSUE TITLE
How did you divide the training set and the test set?

### ISSUE THREAD

**Issue by ttom138** [chenzizhao/biocas-challenge-22#12]
How do you guys divide the training set and test set, like the official division?
![Uploading SPRdsound dataset division.png…]()
